### DISCUSSION TITLE
%EC%A3%BC%EA%B0%84%ED%9A%8C%EA%B3%A0-2024%EB%85%84-5%EC%A3%BC%EC%B0%A8/

### DISCUSSION THREAD

**Discussion by giscus** [heli-os/profile-giscus#6] [Comments]
# %EC%A3%BC%EA%B0%84%ED%9A%8C%EA%B3%A0-2024%EB%85%84-5%EC%A3%BC%EC%B0%A8/

볼타 CTO 테오 블로그

https://dataportal.kr/%EC%A3%BC%EA%B0%84%ED%9A%8C%EA%B3%A0-2024%EB%85%84-5%EC%A3%BC%EC%B0%A8/

**Comment by gosekose** [comment]
처음으로 리뉴얼이라는 큰 기능을 배포할 때 너무 설레고 즐거웠습니다 ㅎㅎ!
정말 훌륭하신 팀원분들과 함께해서 더욱 즐겁고 많이 배웠습니다 !!!
볼타 파이팅!!! 테오, 글렌, 메이, 제이, 셰인 모두 화이팅입니다!
